practice-one/styles/abstracts/variables.css

```diff
@@ -0,0 +1,21 @@
+:root{
+    /* Color */
+    --normal-color: #ff5300;
+    --light-color: #fff7f4;
+    --white: #ffffff;
+    --secondary-color: #7b7b7b;
+    --dark-color: #373737;
+
+    /* Font-large */
+    --bigest: 49px;
```

Comment:
Typo, should be `biggest`. Add this extension to check typo English https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
Thank you for your comment. I will fix it